src/decorators/bootstrap/array.html

```diff
@@ -0,0 +1,32 @@
+<div  class="schema-form-array {{form.htmlClass}}"
```

Comment:
as far as naming these files, can it be src/decorators/schema-form-bootstrap/array.html ?
decorators/bootstrap is a little too generic to be clear what its for